README.md

```diff
@@ -20,6 +20,25 @@ SlowMovie is the code that runs a VSMP on a Raspberry Pi.
 
 SlowMovie requires [Python 3](https://www.python.org). It uses [FFmpeg](https://ffmpeg.org) via [ffmpeg-python](https://github.com/kkroening/ffmpeg-python) for video processing, and [Pillow](https://python-pillow.org) for image processing. [ConfigArgParse](https://github.com/bw2/ConfigArgParse) is used for configuration and argument handling.
 
+### Automated installation
+
+You can quickly install this repository and all required libraries via an install script. This is a simple way to get started if you're not as comfortable with the command line.
+
+To run the install script, open your terminal, copy-paste the following command in, and hit enter.
+
+    bash <(curl https://raw.githubusercontent.com/TomWhitwell/SlowMovie/master/Install/install.sh)
+
+You'll be presented with 4 options when you run the script:
+
+1. **Install/Upgrade SlowMovie** - download the repository and install/update any libraries needed
+2. **Install SlowMovie Service** - run the commands install the SlowMovie service file as described below
+3. **Uninstall SlowMovie Service** - uninstall the SlowMovie service
+4. **Exit**
+
+You can re-run the script at any time in the `Install` folder to update.
+
```

Comment:
```suggestion
You'll be presented with 4 options when you run the script:

1. **Install/Upgrade SlowMovie** - download the repository and install/update any libraries needed
2. **Install SlowMovie Service** - run the commands install the SlowMovie service file as described below
3. **Uninstall SlowMovie Service** - uninstall the SlowMovie service
4. **Exit**

For first-time automated installation, choose 1: Install/Upgrade SlowMovie. When prompted, you can choose "yes" to have the SlowMovie service installed as well which will enable playback to start automatically when the device is powered on or rebooted.

```

Little change to make this a bit more walkthrough-y and removing update instructions to somewhere else.

Reply:
I think the

```
You can re-run the script at any time in the `Install` folder to update.
```

that I took out should be moved to a new subheading Usage -> Updating but it's not letting me suggest down there.

Reply:
I'll also add the other changes you mentioned in separate commit.